crypto-messenger-desktop/src/main/java/cryptomessenger/desktop/infrastructure/client/feign/dialog/DialogMessageDto.java

```diff
@@ -0,0 +1,10 @@
+package cryptomessenger.desktop.infrastructure.client.feign.dialog;
+
+import cryptomessenger.desktop.infrastructure.client.feign.message.MessageDto;
+import lombok.Data;
+
+@Data
+public class DialogMessageDto {
+    private final MessageDto message;
+    private final boolean isMine;
```

Comment:
```suggestion
    private final boolean mine;
```

It is common not to start field name with `is`:
* https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter
* Jackson expects JavaBean naming patterns
* JavaBean naming specification: ![image](https://github.com/KirillSmirnow/crypto-messenger/assets/28138130/d922dabd-bd0c-4820-baa4-f3ebd373e552)